features/register.feature

```diff
@@ -0,0 +1,68 @@
+Feature: A register form so users can sign up to get access to the GOV.UK Sign In demo
+
+  Scenario: User subscribes to mailing list and fills in the form correctly
```

Comment:
Does this duplicate the second to last test?

Reply:
it does, ill remove the last one